File: dodam-system-domain/dodam-domain-rds/src/main/java/b1nd/dodam/domain/rds/club/repository/ClubTimeRepository.java
package b1nd.dodam.domain.rds.club.repository;

import b1nd.dodam.domain.rds.club.entity.ClubTime;
import b1nd.dodam.domain.rds.club.enumeration.ClubTimeType;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ClubTimeRepository extends JpaRepository<ClubTime, ClubTimeType> {
}

File: services/service-club/src/main/kotlin/com/b1nd/dodamdodam/club/domain/club/repository/ClubTimeRepository.kt
package com.b1nd.dodamdodam.club.domain.club.repository

import com.b1nd.dodamdodam.club.domain.club.entity.ClubTimeEntity
import com.b1nd.dodamdodam.club.domain.club.enumeration.ClubTimeType
import org.springframework.data.jpa.repository.JpaRepository

interface ClubTimeRepository : JpaRepository<ClubTimeEntity, ClubTimeType>
